evm/chains/sharding/mainchain_handler/vmc_utils.py

```diff
@@ -0,0 +1,212 @@
+import os
+
+import rlp
+
+from viper import compiler
+
+from eth_abi import (
+    decode_abi,
+    encode_abi,
+)
+
+from eth_keys import KeyAPI
+
+import eth_utils
+
+from evm.utils.address import generate_contract_address
+
+from evm.vm.forks.spurious_dragon.transactions import (
+    SpuriousDragonTransaction,
+    SpuriousDragonUnsignedTransaction,
+)
+
+from evm.chains.sharding.mainchain_handler.config import GASPRICE
+
+sha3 = eth_utils.crypto.keccak
```

Comment:
You can drop the `crypto` and make this just `eth_utils.keccak`

Reply:
cool! no problem